server/parsec/components/memory/account.py

```diff
@@ -17,7 +23,12 @@ async def create(self, email: str) -> None | CreateAccountBadOutcome:
         if email in self._data.accounts:
             return CreateAccountBadOutcome.ACCOUNT_ALREADY_EXISTS
         else:
-            self._data.accounts[email] = MemoryAccount(email)
+            try:
+                email_adapter = TypeAdapter(EmailStr)
```

Comment:
@FirelightFlagboy what do you think ?

Reply:
That what a was expecting, I would move it at the top of the file and name it like a type (`EmailAdapter`)